tests/e2e/Services/Users/UsersBase.php

```diff
@@ -644,6 +644,19 @@ public function testGetUser(array $data): array
         $this->assertIsInt($users['body']['total']);
         $this->assertGreaterThan(0, $users['body']['total']);
 
+        
+        /** 
+          * Test for FAILURE 
+          */ 
+         $response = $this->client->call(Client::METHOD_GET, '/users' . $data['userId'], array_merge([ 
+             'content-type' => 'application/json', 
+             'x-appwrite-project' => $this->getProject()['$id'], 
+         ], $this->getHeaders()), [ 
+             'queries' => ['cursorAfter("unknown")'] 
+         ]); 
+  
+         $this->assertEquals(400, $response['headers']['status-code']); 
```

Comment:
Shouldn't the expected error code be 404?

Reply:
Updated the error code here